docs/ref/models/expressions.txt

```diff
@@ -640,3 +640,28 @@ Let's see how it works::
     Apple: AAPL
     Yahoo: Internet Company
     Django Software Foundation: No Tagline
+
+Adding support for third party backends
+---------------------------------------
+If you are using a backend that uses a different syntax for a certain function,
+you can easily add support for it by monkey patching a new method onto the
+relevant expression class.
+
+Let's say we are using the `Firebird` backend. If `Firebird` would use a
```

Comment:
Oops, just noticed that I think these two occurrences of `Firebird` should either use double-backticks, or no backticks at all. (I'd be inclined to think no backticks at all, since Firebird here isn't a code literal, just a name.)